persistence/src/commonMain/db_global/com/wire/kalium/persistence/ServerConfiguration.sq

```diff
@@ -1,23 +1,42 @@
+import kotlin.Boolean;
+import kotlin.Int;
+
 CREATE TABLE ServerConfiguration (
     id TEXT PRIMARY KEY NOT NULL,
-    title TEXT NOT NULL,
+    title TEXT NOT NULL UNIQUE,
```

Comment:
I know why we are using the title as id in this table, but really I think we need something better later to harden it.

Reply:
Ohh my bad, the title should not be UNIQUE
We can't guaranty the `title` to be unique; otherwise I will be using it as the primary key.
I can change it so that the `title`, `apiBaseUrl` and `webSocketBaseUrl` are unique together

Reply:
https://github.com/wireapp/kalium/pull/489/commits/f4b1d8cc723f64e0dc01de3a5af0176a8cf6aeb0